File: src/main/java/fi/vm/sade/lokalisointi/model/MassUpdateResult.java
package fi.vm.sade.lokalisointi.model;

import lombok.Data;

@Data
public class MassUpdateResult {
  private Integer notModified = 0;
  private Integer created = 0;
  private Integer updated = 0;
  private String status;

  public void incNotModified() {
    notModified++;
  }

  public void incCreated() {
    created++;
  }

  public void incUpdated() {
    updated++;
  }
}

File: src/main/java/fi/vm/sade/lokalisointi/model/MassUpdateResult.kt
package fi.vm.sade.lokalisointi.model

data class MassUpdateResult(
    var notModified: Int? = 0,
    var created: Int? = 0,
    var updated: Int? = 0,
    var status: String? = null
) {
    fun incNotModified() {
        notModified?.inc()
    }

    fun incCreated() {
        created?.inc()
    }

    fun incUpdated() {
        updated?.inc()
    }
}
